src/pages/hegel/reference/becoming/development.mdx

```diff
@@ -0,0 +1,562 @@
+> contradicts itself in itself, because what it unites within itself is
+> self-opposed; but such a union destroys itself (Hegel 2010, 81/21.93-4).
+
+Hegel repeats the point, but sharpens the idea of an internal opposition within
+`becoming` itself. Its movement brings together contradictory impulses and is
+torn asunder by the unrelenting indeterminacy of `pure being` and `nothing`.
+
+`Becoming` is the attempt, one might say, to bring some coherence to the
+conflicting motions of the early categories by carefully noting what must be the
+case: `being` and `nothing` _are_ different, they _are_ the same and they vanish
+into one another. The tools by which `becoming` puts these divergent and
+incompatible ideas, such as moments, movement, coming-to-be and ceasing-to-be,
+are innovative and creative, but equally cannot contain the fury of negativity
+latent in within them. `Becoming` itself thus emerges and perishes.
+
+#### Not Nothing
+
+> This result is a vanishedness, but it is not _nothing_; as such, it would be
+> only a relapse into one of the already sublated determinations and not the
+> result of nothing _and of being_. It is the unity of being and nothing that
+> has become quiescent simplicity. But this quiescent simplicity is _being_, yet
+> no longer for itself but as determination of the whole (Hegel 2010, 81/21.94).
+
+The vanishing of the vanishing would return one to the category `nothing`, but
+Hegel claims that this is not so. The first reason is that this would only put
+the logic back a few steps and _those_ steps would lead precisely _back_ to the
+present circumstance. Now, it may be that the logic merely loops around
+endlessly between `nothing` and `becoming` from this point on, but even this
+loop would be a new result, and one that displays characteristics of stability
+not seen before. The second reason is that this relapse does not take into
+account the complete outcome of `being` and `nothing`, viz. how they are
+inseparable.
+
+It is exactly this inseparability that is left after the rise and fall of
+`becoming`: the vanishing of `being` and `nothing` cannot be coherently
+determined since it itself vanishes, but it remains a fact that these two pure
+categories are deeply entangled and the motion of their vanishing is matched
+only by the inertia of their belonging.
+
+`Being` and `nothing` are thus understood to be simply together, but this
+togetherness itself falls under the header of `being` since this unity _is_. As
+seen above with the specific movements, it is `being` that comes-to-be and
+ceases-to-be. Now a new being must be defined, since the being of `being` and
+`nothing` is no longer pure.
+
+#### Stepping Out
+
+> Becoming, as transition into the unity of being and nothing, a unity which is
+> as existent or has the shape of the one-sided _immediate_ unity of these
+> moments, is _existence_ (Hegel 2010, 81/21.94).
+
+The being of `being` and `nothing` is `existence`. It designates a new immediacy
```

Comment:
Without expressing my clear Miller bias, but I think that determinate being really is a better translation for Dasein than existence. In any case, I think it's worth including here as an alternative translation.

Reply:
Yes, I've added it in as an alternative, but I prefer `existence` since it brings out the idea that this is more a completely new sort of idea rather than a "version of being".